subt/offseter.py

```diff
@@ -0,0 +1,104 @@
+from threading import Thread
+
+class Offseter:
+
+    def __init__(self, config, bus):
+        bus.register("pose3d")
+        self.thread = Thread(target=self.run)
+        self.thread.name = bus.name
+        self.bus = bus
+
+    def start(self):
+        self.thread.start()
+
+    def join(self, timeout=None):
+        self.thread.join(timeout=timeout)
+
+    def run(self):
+        origin = None
+        xyz = None
+        while None in (origin, xyz):
+            dt, channel, data = self.bus.listen()
+            #print("#", channel, data)
```

Comment:
this line is probably forgotten from debug

Reply:
removed in 0e62e32